src/librustc_typeck/check/demand.rs

```diff
@@ -374,6 +383,66 @@ impl<'a, 'gcx, 'tcx> FnCtxt<'a, 'gcx, 'tcx> {
         None
     }
 
+    /// This function checks if the specified expression is a built-in range literal.
+    /// (See: ``librustc::hir::lowering::LoweringContext::lower_expr()``).
```

Comment:
Sorry, I meant single rather than double backticks; I should have clarified. On second thoughts, though, it might be more convenient to include the path:
```rust
/// (See `LoweringContext::lower_expr()` in `src/librustc/hir/lowering.rs`.)
```
as then someone can jump straight to the path. (Like in the following.)
https://github.com/rust-lang/rust/blob/f89622a7548f69c76e5c9857e34be568ba322d02/src/librustc_privacy/lib.rs#L1388